src/Stoa.ts

```diff
@@ -227,7 +231,7 @@ class Stoa {
          * The block data received is stored in the pool
          * and immediately sends a response to Agora
          */
-        this.stoa.post("/block_externalized",
+        this.stoa.post("/block_externalized", IpFilter(white_ip_list, { mode: 'allow' }),
```

Comment:
```suggestion
        this.stoa.post("/block_externalized", IpFilter(white_ip_list, { mode: 'allow', log: false  }),
```
It's okay to remove the logs that are printed on the console.